docs/.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+´Describe the objective of the PR and a brief description of the changes introduced´
+
+
+# Resolves
+
+`Point here the jira issue`
+
+# Impact
+
+`Describe the impact of the PR, wich specifics components of Lumino it affects`
+
+# Changelog
+
+`A bullets list of changes`
+
+```
+1. added foo bar
+2. refactored Foo class
+3. moved foo to bar folder
+```
+
+
+# TODO's
```

Comment:
```suggestion
# TODOs
```

Reply:
this isn't really an acronym, but because it is styled as such, we don't need to add an apostrophe when using the plural form